### ISSUE TITLE
Chart update frequency 

### ISSUE THREAD

**Issue by TangoFan** [jaybo/deepzoom-blog#30]
Can you add information describing the mechanism DeepZoom uses for obtaining and keeping charts updated?

**Comment by jaybo** [comment]
NOAA is undergoing some major revamping of its chart products and distribution channels.  

First there is the switch to a more rational grid system along with switching to the metric system for depths.
https://nauticalcharts.noaa.gov/publications/docs/enc-transformation.pdf

This new system is scheduled for completion in 2024.  Here's the progress to date:
https://distribution.charts.noaa.gov/ENC/rescheme/

Apparently, as part of this effort, the NOAA webpage which I used to obtain the MBTiles containing .pngs of the legacy raster charts has gone away.  So I've avoided this topic for a while since everything seemed to be in flux and haven't updated the charts for a while.

Now a _new_ version of MBTiles from NOAA has popped up with depths only in FEET.  Which is fine, but this new version is only based on vector charts rather than the raster charts traditionally used in DeepZoom.

Vector charts have many virtues, but they have completely lost the artistry found in raster charts which is why I've stubbornly continued to use raster charts in DeepZoom.

**Raster**
Note the sweeping flow of Bellingham and Guemes Channel text, and how multiple font sizes are used for emphasis.
![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/d93e76de-da46-43a4-9ea3-34baa79a90c8)

**Vector**
The labeling sucks!  When the charts have an opacity of say 0.6 and the Mapbox layer appears below to fill in all the missing info, it's (maybe) acceptable.
![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/b87a5c46-59d5-43f8-a5ad-89bbbb7bc0f0)


I was set to switch everything over to vector, but it looks like the available MBTiles are incomplete (eg. Northern Alaska is missing at some zoom levels).  The MBTiles also use a different format (no metadata) compared to the old version, and contain massive numbers of "blank" tiles so I've had to do a bunch of work understanding and dealing with these changes.

So I'm working on this issue, but it's.... complicated.
